docs/EN_US/DataPatterns/DataPatternsMods/DataPatternsBundle.xml

```diff
@@ -1,18 +1,601 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
 "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
 <chapter id="DPBundle">
   <title><emphasis>Data Patterns Bundle</emphasis></title>
 
-  <para>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-  eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-  veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-  commodo consequat.</para>
+  <para>DataPatterns is an ECL bundle that provides additional data profiling
+  and research tools to an ECL programmer.</para>
 
-  <para>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum
-  dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
-  proident, sunt in culpa qui officia deserunt mollit anim id est
-  laborum.</para>
+  <sect1 id="DPBundleInstall">
+    <title>Installation</title>
 
-  <para></para>
+    <para>Depending on which IDE you are using, VS Code or the ECL IDE, follow
+    the appropriate steps below.</para>
+
+    <sect2>
+      <title>Installing in VS Code</title>
+
+      <orderedlist>
+        <listitem>
+          <para>Open VS Code.</para>
+        </listitem>
+
+        <listitem>
+          <para>In the menu bar on the left, click the HPCC Systems
+          logo.</para>
+
+          <para><graphic fileref="../../images/DP001.jpg"/></para>
+        </listitem>
+
+        <listitem>
+          <para>Under “Bundles”, you will see a bundle called
+          “DataPatterns”.</para>
+
+          <para><graphic fileref="../../images/DP002.jpg"/></para>
+        </listitem>
+
+        <listitem>
+          <para>Click on the install link to install the bundle.</para>
+        </listitem>
+      </orderedlist>
+    </sect2>
+
+    <sect2>
+      <title>Installing in the ECL IDE</title>
+
+      <para>You may have to either navigate to the client tools bin directory
+      before executing the command, or use the full path to the ECL
+      tool.</para>
+
+      <orderedlist>
+        <listitem>
+          <para>Use the ECL command line tool to install the bundle.</para>
+
+          <programlisting>ecl bundle install https://github.com/hpcc-systems/DataPatterns.git </programlisting>
+        </listitem>
+
+        <listitem>
```

Comment:
Shouldn't this section be moved to after the ECL IDE installation section, because it is applicable to all IDEs?